docs/DeveloperGuide.md

```diff
@@ -313,7 +313,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use Case: Scheduling a meeting with a new person**
+
+**MSS**
+
+1. User adds new person to FINDvisor
+2. User requests to list persons
```

Comment:
Just a nit but I think we should add punctuation for steps?

Reply:
Sure